### PR TITLE
docker-rmi: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/docker-rmi.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# docker rmi
+
+> Run a command in a new Docker container.
```
Comment (einverne) [inline pages/common/docker-rmi.md:3]: Is this description right?

**Review by einverne — COMMENTED** [review]
@navarroaxel I left a comment for you to take a look.

File: pages/common/docker-rmi.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/docker-rmi.md]: Nop, fixed @einverne !

File: pages/common/docker-rmi.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# docker rmi
+
+> Remove one or more Docker images.
+> More information: <https://docs.docker.com/engine/reference/commandline/rmi/>.
+
+- Show help:
+
+`docker rmi`
+
+- Remove one or more space-separated images:
```
Comment (mebeim) [inline pages/common/docker-rmi.md:10]:
```suggestion
- Remove one or more images given their names:
```
Clearer.

File: pages/common/docker-rmi.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`docker rmi {{image(s)}}`
```
Comment (mebeim) [inline pages/common/docker-rmi.md:12]:
```suggestion
`docker rmi {{image1 image2 ...}}`
```
We usually prefer this for for multiple arguments.

File: pages/common/docker-rmi.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Force remove an image:
+
+`docker rmi --force {{image}}`
+
+- Remove an image without delete untagged parents:
```
Comment (mebeim) [inline pages/common/docker-rmi.md:18]:
```suggestion
- Remove an image without deleting untagged parents:
```
Typo.

**Review by mebeim — CHANGES_REQUESTED** [review]
Thank you @navarroaxel 😁 I've got a couple of suggestions below, other than that looks good.

**Review by mebeim — APPROVED** [review]
Sweet!

**Review by einverne — APPROVED** [review]
Great!

**Pull request merged**